modules/light-clients/08-wasm/types/consensus_state.go

```diff
@@ -24,15 +22,11 @@ func (cs ConsensusState) ClientType() string {
 
 // GetTimestamp returns block time in nanoseconds of the header that created consensus state.
 func (cs ConsensusState) GetTimestamp() uint64 {
-	return cs.Timestamp
+	return 0
```

Comment:
Shouldn't we query the contract in this case?

Reply:
No, the idea is that we will mark this function as deprecated, since it is only used in the light clients, so it doesn't need to be part of the `ConsensusState` interface. In the meantime, until we remove it, we will just return 0.